.drone.yml

```diff
@@ -29,54 +29,54 @@ pipeline:
           go build -v -ldflags "-X main.version=${DRONE_TAG##v} -X main.build=${DRONE_BUILD_NUMBER}" -a -o release/linux/amd64/drone-codecov
         fi
 
-  build_linux_i386:
-    image: golang:1.9
-    pull: true
-    group: build
-    environment:
-      - GOOS=linux
-      - GOARCH=386
-      - CGO_ENABLED=0
-    commands:
-      - |
-        if test "${DRONE_TAG}" = ""; then
-          go build -v -ldflags "-X main.build=${DRONE_BUILD_NUMBER}" -a -o release/linux/i386/drone-codecov
-        else
-          go build -v -ldflags "-X main.version=${DRONE_TAG##v} -X main.build=${DRONE_BUILD_NUMBER}" -a -o release/linux/i386/drone-codecov
-        fi
+  # build_linux_i386:
```

Comment:
Why disable building for i386?

Reply:
Since it requires bash now we need to build everything except linux/amd64 and windows/amd64 on a native agent. The go version missed important features, and instead of reinventing everything within go I'm using the upstream shell script now.

Reply:
Gotcha, i386 is not available.
Thanks.